tasks/image_configuration/downgrade-wpa-hostapd/Kconfig

```diff
@@ -0,0 +1,12 @@
+config DS_DOWNGRADE_WPA_HOSTAPD
+	default "n"
+	bool "Downgrade wpa_supplicant and hostapd to 2.9"
```

Comment:
```suggestion
	bool "Downgrade wpa_supplicant and hostapd to 2.9"
	depends on DS_DISTRO_DEBIAN_12 || DS_DISTRO_UBUNTU_23_04
```

Reply:
As with my previous question. This affects ubuntu 23.04 and 22.04 which are the only two we're shipping. Do you want me to add both of them to that depends statement?

Reply:
Yes, any that it applies to should be listed there.